model.js

```diff
@@ -0,0 +1,84 @@
+const Model = () => {
+    let data = {}
+    const loadFriends = () =>{
+        data["friends"] = [];
+        const pasrse =  $.ajax({
+            method:'get',
+            url:`https://randomuser.me/api/?page=3&results=10`,
+            async: false
+            
+        });
+        const result = JSON.parse(pasrse["responseText"])
+        for(let i = 0 ; i< 7; i++)
+            data["friends"].push({ first : result.results[i]["name"]["first"],last :  result.results[i]["name"]["first"]})
+    } 
+    const loadUserFromApi =() => {
+
+        const pasrse =  $.ajax({
+            method:'get',
+            url:`https://randomuser.me/api/`,
+            async: false
+            
```

Comment:
use the `success` and `error` part of the `.ajax` call. All your data handling should be in the `success` section. Look back at the lesson in the kernel